tests/unit/src/test/resources/foldingRange/expect/Blocks.scala

```diff
@@ -17,23 +17,23 @@ class A >>region>>{
 
   def longSingleLineBlock = { ???; ???; ???; }
 
-  def chain = >>region>>Seq(1).map>>region>>{
-    x =>
+  def chain = >>region>>Seq(1).map{
+    >>region>>x =>
```

Comment:
So this will work better and is actually a fix. Who fixed this you ask? I have no idea :sweat_smile:

Reply:
Great? haha